include/aikido/planner/ompl/Planner.hpp

```diff
@@ -209,6 +210,51 @@ trajectory::InterpolatedPtr planCRRTConnect(
     double _minStepsize,
     double _minTreeConnectionDistance);
 
+/// Use LRA* planner to plan a path that moves from the
+/// start to a goal region given a graph to search on
+/// \param _start The start state
+/// \param _goal The goal state
+/// \param _statespace The StateSpace that the planner must plan within
+/// \param _interpolator An Interpolator defined on the StateSpace. This is used
+/// to interpolate between two points within the space.
+/// \param _dmetric A valid distance metric defined on the StateSpace
+/// \param _sampler A Sampleable that can sample states from the
+/// StateSpace. Warning: Many OMPL planners internally assume this sampler
+/// samples uniformly. Care should be taken when using a non-uniform sampler.
```

Comment:
This warning only makes sense for `planOMPL`. You should either reword to rephrase it specifically for LRA* or remove it entirely.